conf/production.ini

```diff
@@ -1,7 +1,6 @@
 [pipeline:main]
 pipeline:
   proxy-prefix
-  raven
   lms
```

Comment:
All this raven and sentry stuff in the config file is no longer needed, sentry-sdk's Pyramid integration does it all for us.